tools/python/python-3.10.1.yaml

```diff
@@ -0,0 +1,68 @@
+# metadata for python
+info:
+  id: tools/languages/python
+  version: 3.10.1
+  priority: 10
+
+demands:
+  windows and x64:
+    unless:  # look for a system installed version first
+      from: [$host.PATH, $host.ProgramFiles/*python* ,c:/python]                      # paths to search
+      where: [ python.exe,py3.exe, python3.exe,  py.exe  ]        # one of these choices
+      run: $0 --version                                              # run the choice with this cmdline
+      select: /Python (3.8|3.9|3.\d\d+)\.\d*/g                        # parse the output with this regex
+      is: "$1 == '3.10'"                                                # this expression must be true.

```

Comment:
Is this matching Python 3.8 - 3.10 or only 3.10? The error message implies 3.8+, but this logic looks like it's only 3.10.